docs/docs/react-without-jsx.md

```diff
@@ -4,11 +4,13 @@ title: React Without JSX
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+JSX is not a requirement for using React. React exposes an API for creating elements with functions instead of JSX. This is especially convenient if you want to use React without introducing compilation steps to your build process.
 
-Each JSX element is just syntactic sugar for calling `React.createElement(component, props, ...children)`. So, anything you can do with JSX can also be done with just plain JavaScript.
+### Using `React.createElement()`
 
-For example, this code written with JSX:
+Each JSX element is actually syntactic sugar for calling `React.createElement(component, props, ...children)`. Therefore, creating elements without JSX is possible by using `React.createElement()`.
+
+For example, the code below written with JSX:
```

Comment:
Maybe

>Consider the code below written with JSX:

?

Otherwise it doesn't read like a sentence, but the continuation is already a separate sentence. So it's a bit awkward.